sources/lib/run-time/llvm-runtime.h

```diff
@@ -18,7 +18,7 @@ typedef float                   DSFLT;
 typedef double                  DDFLT;
 
 #define ITAG 1
-#define I(n) ((dylan_value) (((intptr_t)(n) << 2) | ITAG))
+#define I(n) ((dylan_value) (((unsigned long)(n) << 2) | ITAG))
```

Comment:
Actually I do think it might be good have an 'audit' of what C types we use here and elsewhere to check it's all sound with respect to 32 vs 64 bit. And maybe use the `<stdint.h>` types more?

Reply:
Agree that moving towards more `<stdint.h>` types would be a good thing.